CONTRIBUTING-ACL2020.md

```diff
@@ -6,42 +6,42 @@ Notes: see [CONTRIBUTING.md](./CONTRIBUTING.md) for general instructions.
 
 Before you start, please make sure you have a GitHub account and have your ssh key added to your GitHub account. Also, Python should be installed into your device to setup the environment.
 
-### Get a GitHub Repository
+### Get the GitHub Repository
+You can either do it with git in the command line, or via the website.
 
+#### Option 1. Via command line and git
 ```shell
 cd /path/to/the/local/folder
 git clone git@github.com:acl-org/acl-2020-virtual-conference.git
```

Comment:
Or probably can just use `git clone --recursive git@github.com:acl-org/acl-2020-virtual-conference.git` maybe? and can explain submodule after that?

Reply:
I think it is important for people to understand that there are 2 imbricated repositories. 
The "double init" makes it clearer (because the people have to go through both steps - it also forces people to actually read the readme instead of just skimming through).
What do you think?

Reply:
Sure, sounds good. Do you think mentioning how to bump sitedata might also be helpful?

Reply:
Good idea! Can you add it in "How to Create a Pull Request for the Website `acl-2020-virtual-conference` AND the Data `sitedata_acl2020`" ?

Reply:
Actually this is done, sorry I did not notice this before!